apps/condo/domains/billing/schema/BillingReceipt.js

```diff
@@ -90,6 +89,56 @@ const BillingReceipt = new GQLListSchema('BillingReceipt', {
             isRequired: true,
         },
 
+        formula: {
+            schemaDoc: 'Calculation formula. Example: balance + charge + recalculation + privilege + penalty',
+            type: Text,
+            // TODO(DOMA-6519): change read-only access to { create: true, read: true, update: true } after toPayDetails field removal
+            access: access.readOnlyAccess,
+        },
+
+        charge: {
+            ...MONEY_AMOUNT_FIELD,
+            schemaDoc: 'Amount of money charged by paid period. Example: "50.00", "-50.00"',
+            // TODO(DOMA-6519): change read-only access to { create: true, read: true, update: true } after toPayDetails field removal
+            access: access.readOnlyAccess,
+        },
+
+        balance: {
+            ...MONEY_AMOUNT_FIELD,
+            schemaDoc: 'Recipient balance on the receipt creation moment. Example: "50.00", "-50.00"',
+            // TODO(DOMA-6519): change read-only access to { create: true, read: true, update: true } after toPayDetails field removal
+            access: access.readOnlyAccess,
+        },
+
+        recalculation: {
+            ...MONEY_AMOUNT_FIELD,
+            schemaDoc: 'Recipient balance recalculation in case of overpaid or etc. Example: "50.00", "-50.00"',
```

Comment:
It’s more about cases then you create a wrong receipt or have a mistake in previous receipts.